hack/local-up-cluster.sh

```diff
@@ -504,6 +510,26 @@ EOF
     fi
 }
 
+# start_discovery relies on certificates created by start_apiserver
+function start_discovery {
+    # TODO generate serving certificates
+    
+    DISCOVERY_SERVER_LOG=/tmp/kubernetes-discovery.log
+    ${CONTROLPLANE_SUDO} "${GO_OUT}/kubernetes-discovery" \
+      --cert-dir="${CERT_DIR}" \
+      --client-ca-file="${CERT_DIR}/client-ca-bundle.crt" \
+      --bind-address="${API_BIND_ADDR}" \
+      --secure-port="${DISCOVERY_SECURE_PORT}" \
+      --tls-ca-file="${ROOT_CA_FILE}" \
+      --etcd-servers="http://${ETCD_HOST}:${ETCD_PORT}"  >"${DISCOVERY_SERVER_LOG}" 2>&1 &
+    DISCOVERY_PID=$!
+
+    # Wait for kubernetes-discovery to come up before launching the rest of the components.
```

Comment:
> why?

Makes it easier to fail on it.  This is just to start it up, but I'll need to get this up in a pod later, with kube proxy pointing to its service, and then I'd like to generate the bootstrap kubeconfig to point to it.